jwt-authorizer/templates/jwt-authorizer-configmap.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jwt-authorizer-config
+data:
+  authorizer.yaml: |
+    global:
+      REALM: {{ .Values.host }}
+    
+      # Options are Bearer, Basic, none
+      WWW_AUTHENTICATE: Basic
+      LOGLEVEL: {{ .Values.jwt_authorizer.loglevel | default "DEBUG" }}
+    
+      # Authorizer Configurations
+      # NO_AUTHORIZE: True
+    
+      # Resource Checkers
+      RESOURCE_CHECKS_DEFAULT: 
+        - group_membership
+    
+      # group_membership options
+      GROUP_MAPPING:
+        {{ range $key, $value := .Values.jwt_authorizer.group_mapping }}
+        {{ $key | quote }}: {{ $value }}
+        {{- end }}
+    
+      OAUTH2_JWT:
+        ISS: https://{{ .Values.host }}
+        KEY_ID: reissuer
+        AUD:
+          DEFAULT: https://{{ .Values.host }}
+          INTERNAL: https://{{ .Values.host }}/api
+        KEY_FILE: {{ .Values.signing_key_file | default "/etc/jwt-authorizer/secrets/signing_key.pem" }}
```

Comment:
Since these look like paths in a container, will these ever be different?  It seems like this is dependent on how you mount the volume containing the secrets?  You don't allow configuration of the path of the volume mount on jwt-authorizer-deployment.yml:29

Reply:
fixed